HIP-0002.md

```diff
@@ -35,6 +35,10 @@ This proposal doesn't define if the content served should be static (file in the
 
 Requests should be made using only HTTPS. Handshake's HTTPS requests are secured by DANE & PoW.
 
+In implementations that allow sending to both addresses and domain aliases, measures must be taken to disambiguate between addresses and domains of the same length. If sharing an entry field, aliases should be prefixed with an `@` symbol. 
+
+For example, `hs1qshuyulxra3pqpwr40303t8pn79232zztuk4qgz` should be interpreted as a wallet address, while `@hs1qshuyulxra3pqpwr40303t8pn79232zztuk4qgz` should be interpreted as a domain alias with wallet address located at `hs1qshuyulxra3pqpwr40303t8pn79232zztuk4qgz/.well-known/wallets/HNS`.
```

Comment:
IMO, domain names that are valid addresses should simply fail and we should refuse to resolve them. Why not?

Reply:
Agreed.

Reply:
Approving as it is, besides the comment, because this how the protocol is currently used (Bob Wallet)